src/test/java/org/dcsa/bkg/model/transferobjects/BookingTOTest.java

```diff
@@ -0,0 +1,337 @@
+package org.dcsa.bkg.model.transferobjects;
+
+import com.fasterxml.jackson.core.JsonProcessingException;
+import com.fasterxml.jackson.databind.JsonNode;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.datatype.jsr310.JavaTimeModule;
+import org.dcsa.core.events.model.enums.CargoGrossWeight;
+import org.dcsa.core.events.model.enums.CargoMovementType;
+import org.dcsa.core.events.model.enums.CommunicationChannel;
+import org.dcsa.core.events.model.enums.ReceiptDeliveryType;
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.BeforeEach;
+import org.junit.jupiter.api.DisplayName;
+import org.junit.jupiter.api.Test;
+
+import javax.validation.ConstraintViolation;
+import javax.validation.Validation;
+import javax.validation.Validator;
+import javax.validation.ValidatorFactory;
+import java.text.SimpleDateFormat;
+import java.time.OffsetDateTime;
+import java.util.Collections;
+import java.util.Set;
+
+@DisplayName("Tests for BookingTO")
+class BookingTOTest {
+
+  private Validator validator;
+  private ObjectMapper objectMapper;
+  private BookingTO validBookingTO;
+
+  @BeforeEach
+  void init() {
+    ValidatorFactory factory = Validation.buildDefaultValidatorFactory();
+    validator = factory.getValidator();
+
+    objectMapper = new ObjectMapper().registerModule(new JavaTimeModule());
```

Comment:
In DCSA core, we use the following definition for ObjectMapper:

```java
    private final ObjectMapper objectMapper = new ObjectMapper()
            .registerModule(new JavaTimeModule()) // Needed to process time objects.
            .disable(SerializationFeature.WRITE_DATES_AS_TIMESTAMPS)
            .disable(DeserializationFeature.ADJUST_DATES_TO_CONTEXT_TIME_ZONE)
            .enable(SerializationFeature.WRITE_DATES_WITH_ZONE_ID);
```

Are we sure we do not need some of these `.enable` cases here ?